symengine: 0.7.0 -> 0.8.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/symengine/versions.


meta.description for symengine is: "A fast symbolic manipulation library"

meta.homepage for symengine is: "https://github.com/symengine/symengine"

meta.changelog for symengine is: ""


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/symengine/symengine/releases/tag/v0.8.1)

- [Compare changes on GitHub](https://github.com/symengine/symengine/compare/v0.7.0...v0.8.1)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 0.8.1 with grep in /nix/store/wnn1s3v2kzs0l34b1q81nvdy4xbpxas1-symengine-0.8.1
- found 0.8.1 in filename of file in /nix/store/wnn1s3v2kzs0l34b1q81nvdy4xbpxas1-symengine-0.8.1
- directory tree listing: https://gist.github.com/aca93f709f52c573e97b8392be9bbe29

- du listing: https://gist.github.com/02542d82325128716000d49546da9078

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
12 total rebuild path(s)

3 package rebuild(s)

3 x86_64-linux rebuild(s)
3 i686-linux rebuild(s)
3 x86_64-darwin rebuild(s)
3 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
python38Packages.symengine
python39Packages.symengine
symengine
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/wnn1s3v2kzs0l34b1q81nvdy4xbpxas1-symengine-0.8.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A symengine https://github.com/r-ryantm/nixpkgs/archive/d7c262d057cdeb25290405c261176fc7dd6d5efb.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/wnn1s3v2kzs0l34b1q81nvdy4xbpxas1-symengine-0.8.1
ls -la /nix/store/wnn1s3v2kzs0l34b1q81nvdy4xbpxas1-symengine-0.8.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python38Packages.symengine</li>
    <li>python39Packages.symengine</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>symengine</li>
  </ul>
</details>

---

###### Maintainer pings

cc @costrouc for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
symengine: 0.7.0 -> 0.8.1

r-rmcgibbo commented:
Result of `nixpkgs-review pr 137709` at d7c262d0 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/d7ef2b17690a57dab643ad49d17d836a">python38Packages.symengine</a></li>
    <li><a href="https://gist.github.com/5d83720a12c2ccaa09e47ccbd99b5671">python39Packages.symengine</a></li>
  </ul>
</details>
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>symengine</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

risicle commented:
Might need a simultaneous bump of the python package.

risicle committed:
python3Packages.symengine: 0.7.2 -> 0.8.1

patching out cython version requirement as their choice seems to have
been arbitrary ("the latest")

risicle commented:
`nixpkgs-review` now happy for me, macos 10.15 & nixos x86_64

bobby285271 commented:
Result of `nixpkgs-review pr 137709` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python38Packages.symengine</li>
    <li>python39Packages.symengine</li>
    <li>symengine</li>
  </ul>
</details>
